vscodium@trymeouteh/vscodium@trymeouteh.nemo_action.in

```diff
@@ -0,0 +1,7 @@
+[Nemo Action]
+Name=Open in VSCodium
+Comment=Open VSCodium in the selected folder
+Exec=codium %F
```

Comment:
Take a look at the Action for VSCode. It uses `%P` instead of `%F`. Nemo already has a handle for `%F`.